sdk/storage/azure-storage-file-share/azure/storage/fileshare/_share_client.py

```diff
@@ -6,40 +6,46 @@
 # pylint: disable=docstring-keyword-should-match-keyword-only, too-many-lines
 
 from typing import (
-    Any, Dict, Iterable, Literal, Optional, Union,
+    Any, cast, Dict, Literal, Optional, Union,
     TYPE_CHECKING
 )
-from typing_extensions import Self
+from typing_extensions import Never, Self
 
 from azure.core.exceptions import HttpResponseError
-from azure.core.tracing.decorator import distributed_trace
+from azure.core.paging import ItemPaged
 from azure.core.pipeline import Pipeline
+from azure.core.tracing.decorator import distributed_trace
 from ._deserialize import deserialize_permission, deserialize_share_properties
 from ._directory_client import ShareDirectoryClient
 from ._file_client import ShareFileClient
 from ._generated import AzureFileStorage
 from ._generated.models import (
     DeleteSnapshotsOptionType,
-    SignedIdentifier)
+    ShareStats,
+    SignedIdentifier
+)
 from ._lease import ShareLeaseClient
 from ._models import ShareProtocols
+from ._parser import _parse_snapshot
 from ._serialize import get_api_version, get_access_conditions
```

Comment:
I think we also do alphabetical order for the imports, so 'get_access_conditions" should come before 'get_api_version'

Reply:
Oops, fixed in `_share_client_async.py` but forgot to here :(